docs/intro/overrides.rst

```diff
@@ -3,13 +3,17 @@
 Overrides
 =========
 
-Overrides contains mapping rules to associate which URLs a particular
-Page Object would be used. The URL matching rules is handled by another library
-called `url-matcher <https://url-matcher.readthedocs.io>`_.
+Overrides are simply rules represented by a list of :class:`~.ApplyRule` which
+associates which URL patterns a particular Page Object would be used. The URL
+matching rules is handled by another library called
+`url-matcher <https://url-matcher.readthedocs.io>`_.
 
 Using such rules establishes the core concept of Overrides wherein a developer
-could declare that a specific Page Object must be used *(instead of another)*
-for a given set of URL patterns.
+could declare that for a given set of URL patterns a specific Page Object must
+be used:
+
+    * instead of another Page Object, or 
```

Comment:
I think only this use case counts as "Override"

Reply:
Updated in https://github.com/scrapinghub/web-poet/pull/84/commits/52a6f474bd97d1626eaf153241496a7a78c40df6 where the focus shifts into matching the URL pattern and `to_return` class type.